service_management/azure/test/integration/affinity_group/Affinity_test.rb

```diff
@@ -1,55 +0,0 @@
-#-------------------------------------------------------------------------
-# Copyright 2013 Microsoft Open Technologies, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#--------------------------------------------------------------------------
-require 'integration/test_helper'
-
-describe Azure::BaseManagementService do
```

Comment:
This file contained two tests. 1. Creating valid affinity group & 2. Creating invalid affinity group. The first one was already covered in other tests & I moved the other one in the Create_Affinity_test.rb line 73. So that we can eliminate the file.